src/vivarium_csu_alzheimers/model_specifications/model_spec.yaml

```diff
@@ -57,8 +58,13 @@ configuration:
         bbbm_test_count:
             include:
                 - 'bbbm_test_results'
-        bbbm_test_eligibility:
+                - 'semester'
+        bbbm_test_newly_eligible:
             include:
                 - 'bbbm_test_results'
+        bbbm_test_eligible: 
+            include:
+                - 'alzheimers_disease_and_other_dementias'
```

Comment:
This one doesn't need the alzheimer's state stratification, because only people actually in BBBM are eligible. We need it for the observations you'll create in the next PR, because people could have previously been eligible, but now be in a different alzheimer's state.